ngr_spider/ogc_api_tiles.py

```diff
@@ -86,66 +86,54 @@ def __init__(self, url):
         self.service_url = url
         self.__load_landing_page(url)
 
-    # https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/#vector
     def get_layers(self):
-        service_layer_name: str
-        service_layer_title: str = ""
-        service_layer_abstract: str
-        service_layer_crs: str = ""
-        service_layer_min_scale: str = ""
-        service_layer_max_scale: str = ""
-        service_data_type: str
-
-        # process styles
-        # TODO style should be generated based on the type of the tiles; png, Vector etc.
-        vector_tile_styles: list[VectorTileStyle] = self.get_styles()
+        tiles_title: str
+        tiles_abstract: str
+#We doen in de viewer nog niets met de verschillende tilesets
```

Comment:
Deze regels zijn opgenomen in models.py als OatTileSet(), dus deze comment zou weg kunnen